cea/technologies/solar/photovoltaic.py

```
@@ -89,7 +89,7 @@ def calc_PV(locator, config, latitude, longitude, weather_data, datetime_local,
     solar_properties = solar_equations.calc_sun_properties(latitude, longitude, weather_data, datetime_local, config)
 
     # calculate properties of PV panel
-    panel_properties_PV = get_properties_PV_db(locator.get_db4_components_conversion_conversion_technology_csv('PHOTOVOLTAIC_PANELS'), config)
+    panel_properties_PV = calc_properties_PV_db(locator.get_db4_components_conversion_conversion_technology_csv('PHOTOVOLTAIC_PANELS'), type_PVpanel)
```

Ruff (0.8.2)

Undefined name `calc_properties_PV_db`

(F821)